jtl/functions.go

```diff
@@ -1214,7 +1214,7 @@ func fnTenant(ctx Context, doc *JDoc, params []string) interface{} {
 	if h.TenantId == "_default" {
 		//The default tenant handler is used. We need to drill-down into context to find out the actual tenant
 		tenantHeaderKey := GetConfig(ctx).HttpTenantHeader
-		if ctx.Value(tenantHeaderKey) != "" {
+		if ctx.Value(tenantHeaderKey) != nil {
```

Comment:
Should we check for both the empty string and nil? Also, this reminds me, we should do the same in cron too. I think currently, it only check for empty string.